source/views/sis/course-search/index.js

```diff
@@ -47,13 +50,10 @@ class CourseSearchView extends React.PureComponent<Props, State> {
 
 	state = {
 		searchResults: [],
-    searchActive: false,
-	}
-
-	componentWillMount() {
-		this.headerOpacity = new Animated.Value(1)
-		this.searchBarTop = new Animated.Value(71)
-		this.containerHeight = new Animated.Value(125)
+		searchActive: false,
+		headerOpacity: new Animated.Value(1),
+		searchBarTop: new Animated.Value(71),
+		containerHeight: new Animated.Value(125),
```

Comment:
I… er… do these belong in `state`? Honest question; I don't know.

Reply:
I had them as regular member variables and either eslint or flow got mad at me..can't remember which!

Reply:
It may want you to define them when you put them on the class, but this should work, I think:

```js
class foo {
    headerOpacity = new Animated.Value(1)
}
```